enterprise/templates/enterprise/enterprise_customer_login_page.html

```diff
@@ -13,7 +13,7 @@
     <div class="enterprise-container">
       <div class="enterprise-login-container">
 
-          <h2 class="enterprise-login-title">{{ enterprise_login_title_message|safe }}</h2>
+          <h2 class="h2 enterprise-login-title">{{ enterprise_login_title_message|safe }}</h2>
```

Comment:
nit: Shouldn't need the .h2 class if the element is already an h2 element?

Reply:
I think it does, for the bootstrap styles? But I originally added it because I had switched this to an h1, not realizing there was an h1 in the header. I'll look into taking it out.

Reply:
Bootstrap styles both the heading elements (e.g., `<h2>`) and the heading classes (e.g., `.h2`) 😄